electrophysiology/models.py

```diff
@@ -40,15 +42,23 @@ class ExtracellularRecording(BaseExperimentalData):
     impedances = models.ForeignKey(Dataset, blank=True, null=True, related_name="extracellular_impedances",
                                    help_text="binary array for measured impedance of each channel (ohms).")
     amplifier = models.ForeignKey(Amplifier, blank=True, null=True,
-                                  help_text="The amplifier used in this recording. TODO: multiple amplifiers?")
+                                  help_text="The amplifier used in this recording.")
+								   # TODO: multiple amplifiers? 
+								   # they would have their own ExtracellularRecording objects in that case.
```

Comment:
If you have multiple shanks all connected to the same amplifier (in different locations) do they each get an object - and mandatorily have to be split into separate files?